global/css/_global.css

```diff
@@ -204,21 +204,43 @@ svg {
 
 .component-slider-single.component-slider-peek .swiper-slide-prev > *,
 .component-slider-single.component-slider-peek .swiper-slide-next > * {
-  @apply su-opacity-[.27] md:su-opacity-100 su-pointer-events-none md:su-pointer-events-auto;
+  @apply su-opacity-25 md:su-opacity-100 su-pointer-events-none md:su-pointer-events-auto;
+}
+
+.component-slider-single.component-slider-peek.component-slider-vertical-videos .swiper-slide-prev > *,
+.component-slider-single.component-slider-peek.component-slider-vertical-videos .swiper-slide-next > * {
+  @apply su-opacity-25 su-pointer-events-none;
+}
+
+.su-slider-dark .component-slider-single.component-slider-peek.component-slider-vertical-videos .swiper-slide-prev > *,
+.su-slider-dark .component-slider-single.component-slider-peek.component-slider-vertical-videos .swiper-slide-next > * {
+  @apply su-opacity-40 su-pointer-events-none;
 }
 
 .component-slider .component-slider-pagination {
   @apply su-static su-flex su-gap-10 su-text-left;
 }
 
 .component-slider .swiper-pagination-bullet {
-  @apply su-mx-0 su-transition su-opacity-100 su-w-20 su-h-20 su-bg-transparent su-border-2 su-border-solid su-border-black-30 dark:su-border-black-60;
+  @apply su-mx-0 su-transition su-opacity-100 su-w-20 su-h-20 su-bg-transparent su-border-2 su-border-solid su-border-black-40 dark:su-border-black-60;
```

Comment:
For all light mode carousels, I updated the pagination bullet border from black-30 to black-40 since they're also used on the Campaign page - which has a slightly darker light mode background color (campaign-fog-light). The black-40 would make it more accessible.